docs/driver.md

```diff
@@ -28,4 +28,16 @@ By default, the driver launches the IDE in the non-headless mode. Note, that the
 
 #### Virtual Machine options
 `driver.vmOptions = ["-Xmx4096m"]`
-Empty by default. Used by the driver to further customize the IDE. 
\ No newline at end of file
+Empty by default. Used by the driver to further customize the IDE.
+
+#### Screen size
+```
+driver {
+  screen {
+    width = 1920,
```

Comment:
Commas are redundant between keys and typically skipped

Reply:
HOCON is a strict superset of JSON, hence they're allowed in the first place; compare https://virtuslab.github.io/tips/#other/yaml-vs-json